src/app/containers/MediaPageMain/index.jsx

```diff
@@ -27,12 +52,9 @@ const MediaPageMain = props => {
       <main role="main">
         <Grid>
           <GridItemConstrainedMedium>
-            <Headline script={script} service={service}>
-              {title}
-            </Headline>
-            <Paragraph script={script} service={service}>
-              {subtitle}
-            </Paragraph>
+            {blocks.map(block => {
+              return renderBlock({ block, script, service, key: block.text });
```

Comment:
Is block.text always going to be unique?

Reply:
As above, this is temporary work to stop us hardcoding the heading and paragraph values to block[0] and block[1].